app/javascript/controllers/trello_controller.js

```diff
@@ -0,0 +1,56 @@
+import { Controller } from 'stimulus';
+import Sortable from 'sortablejs';
+
+export default class extends Controller {
+  static values = { columnId: String }
+
+  connect() {
+    this.sortable = Sortable.create(this.element, {
+      animation: 150,
+      group: {
+        name: 'shared',
+        pull: true,
+        sort: true,
+      },
+      filter: '.js-card--sticky',
+      fallbackOnBody: true,
+      swapThreshold: 0.65,
+      onEnd: this.reorder.bind(this),
```

Comment:
I think you don't need to bind if you use an arrow function for the reorder func 😄